lib/analyzer/package-managers/apt.ts

```diff
@@ -22,6 +22,24 @@ export function analyze(
   });
 }
 
+export function analyzeDistroless(
```

Comment:
I assume that's a material for another PR, but I find it weird that this function is not `async` and we return a `Promise.resolve`.. we not just having it `async` and return the value without the `promise.resolve`

Reply:
we're not ```await```ing on anything, so we don't need it to be ```async```.
this function returns a Promise as all other analyzer functions return promises, because the base code ```await``` on a list of functions, so we want all of them to be promises.
I'm more than happy to have a bigger refactor in these areas after we remove the dynamic scanning bits, I'm very unhappy with how the code looks right now.